### PR TITLE
fix: Use list pull request files api

### PR DESCRIPTION
**Pull request by levsa**
and sum the changes for each file instead of parsing diff to solve the issue with max 300 files error from rest api.

Probably also solves #53 ?

### CODE REVIEW

**Review by subhamg — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -168,15 +167,16 @@ async function readFile(path) {
...
 
-function getChangedLines(isIgnored, diff) {
-  return Diff.parsePatch(diff)
-    .flatMap(file =>
-      isIgnored(file.oldFileName) && isIgnored(file.newFileName)
-        ? []
-        : file.hunks
+function getChangedLines(isIgnored, pullRequestFiles) {
+  console.log(typeof(pullRequestFiles));
+  console.log(pullRequestFiles);
```
Comment (levsa) [inline index.js:172]: I'll remove these leftovers.

**Review by joellidin — APPROVED** [review]

**Comment by yahel2410** [comment]
Hey, what's the status here? can it be merged?

**Comment by pascalgn** [comment]
Fixes #53

**Comment by pascalgn** [comment]
Thanks for the PR, it's merged now! 👍

**Pull request merged**